### PR TITLE
Home-page

### PR DESCRIPTION
**Pull request by SaurabhChawda**
I have added a home page in Quizilla (A Quiz App).
It Includes;-
1. Navigation Component
2. Header Component
3. Search Component
4. Navfor Tab Component.
Screenshot (319)

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,42 @@
...
+:root {
+  --primary-color: #003459;
+  --secondary-color: #00243d;
+  --white-color: white;
+  --black-color: black;
+  --font-size-XX-large: clamp(3rem, 12vw, 10rem);
+  --font-size-X-large: clamp(1.2rem, 2.5vw, 10rem);
+  --font-size-large: clamp(1rem, 1.5vw, 10rem);
+  --font-size-regular: clamp(12px, 2.5vw, 1rem);
+  --font-size-small: clamp(8px, 1vw, 1rem);
```
Comment (JantuDeb) [inline src/App.css:28]: This is really good that you are using `clamp`

File: src/components/Header/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+          <NavLink to="/">
+            <img
+              className="nav-bar__img--logo"
+              src="/assets/image/HomePage-Images/Quizella.png"
+              alt="Trendy-T"
+            />
+          </NavLink>
+        </div>
+        <div className="nav-bar--serach">{value && <SearchBar />}</div>
+      </div>
```
Comment (JantuDeb) [inline src/components/Header/Header.jsx:19]: You can return this `<div></div>` instead of `<></>`

File: src/components/NavForTab/NavForTab.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+            alt="Report"
+          />
+        </NavLink>
+        <NavLink to="/login/">
+          <img
+            className="nav-bar-tab--icon"
+            src="/assets/image/HomePage-Images/Login.png"
+            alt="Login"
+          />
+        </NavLink>
```
Comment (JantuDeb) [inline src/components/NavForTab/NavForTab.jsx]: `NavLink` is use for style active links so if you do not want to use active style then use `<Link>`

**Review by JantuDeb — APPROVED** [review]
left few comments

File: src/Pages/Home-Page/Home.jsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import { Nav } from "../../components/Nav/Nav";
+// import { Footer } from "../../components/Footer/Footer";
```
Comment (Bharatrathod-81) [inline src/Pages/Home-Page/Home.jsx:2]: comment can be removed

**Review by Bharatrathod-81 — APPROVED** [review]

File: src/App.css (hunk 1 above)
Comment (SaurabhChawda) [reply src/App.css:28]: ThankYou

File: src/components/NavForTab/NavForTab.jsx (hunk 3 above)
Comment (SaurabhChawda) [reply src/components/NavForTab/NavForTab.jsx]: updated,Thankyou

File: src/Pages/Home-Page/Home.jsx (hunk 4 above)
Comment (SaurabhChawda) [reply src/Pages/Home-Page/Home.jsx:2]: Okay,I will removed,ThankYou

**Pull request merged**